vms/platformvm/state/state.go

```diff
@@ -178,6 +179,12 @@ type State interface {
 	Close() error
 }
 
+// TODO: Remove after v1.12.x is activated
+type stateBlk struct {
```

Comment:
nit: can we add more detail about the history of this struct (i.e. legacy format that some long-lived dbs have)